handbook/engineering/security/index.md

```diff
@@ -4,9 +4,51 @@ We think that security is an enabler for the business. Sourcegraph is committed
 
 ## Goals
 
-Customers know they can confidently and securely use Sourcegraph with their private code.
+### Long term
 
-[This (internal) document](https://docs.google.com/document/d/1HhYfnkC17yCPqqw91H1QAF7q8toOOiNtoOOw06hqrNQ/) details our plan.
+Every organization, is confident they can trust Sourcegraph with the security of their Source code.
+
+### Medium term
+
+The goals below lie on the path to our long-term goals, and represent the next 3-6 months of work.  These goals will be delivered serially, but may be worked on in parallel.
+
+**We share our private [infrastructure repository](http://github.com/sourcegraph/infrastructure) on Sourcegraph cloud**
```

Comment:
```suggestion
**We store our private [infrastructure repository](http://github.com/sourcegraph/infrastructure) on Sourcegraph cloud**
```
Assuming you mean store or index here, since we still want it to be private

Reply:
Updated to host in latest.